[Translation] tutorial/part-eight

Andrewnt219 opened the pull request:
[Translation] tutorial/part-eight

Andrewnt219 committed:
Up to line 155

Andrewnt219 committed:
155 finished

Andrewnt219 committed:
New laptop

Andrewnt219 committed:
Up to line 334

Andrewnt219 committed:
Fin

Andrewnt219 committed:
Fin 2

Andrewnt219 committed:
fix minor typo

vitokhangnguyen reviewed (changes requested):
Chào bạn, sau khi nhìn lướt qua, mình có 1 số góp ý sau, bạn xem qua giúp mình nhé:

* Title của PR bạn đặt theo cú pháp: `[Translation] <TÊN_DOCUMENT_TRONG_ISSUE_TRACKER>` giúp mình nhé. Mình fix lại giúp bạn rồi.
* Mình thấy là trong thư mục `docs/tutorial/part-six` có xuất hiện thêm file `mục lục.md`. Bạn cho mình hỏi file này là file gì được không? Nếu không cần thiết bạn xóa đi giúp mình. Khi bạn translate, tránh ảnh hưởng đến những file không liên quan khác nhé.
* Ở đầu mỗi bài dịch thường có vài dòng manifest, ví dụ:
    ```
    ---
    title: Preparing a Site to Go Live
    typora-copy-images-to: ./
    disableTableOfContents: true
    ---
    ```
    Phần này bạn dịch mỗi nội dung tiêu đề, ở đây là `Preparing a Site to Go Live` thôi nhé. Những chỗ khác giữ nguyên vì đây là phần khai báo quan trọng khi host tài liệu lên web chính thức.

Bạn fix những chỗ này trước giúp mình nhé. Mình sẽ xem qua kĩ hơn và có gì báo thêm cho bạn.

Thank you!

Andrewnt219 committed:
Update after revision

Reverting changes to "component"
Reverting changes to comments
Translating "Gatsby site" -> "trang web Gatsby"
Fixing manifest translation
Deleting tutorial/part-six/"mục lục.md"

Andrewnt219 commented:
@vitokhangnguyen 
Removing translation of "component"
Removing translation of comments
Translating "Gatsby site" -> "trang web Gatsby"
Fixing manifest translation
Deleting tutorial/part-six/"mục lục.md"

Andrewnt219 committed:
translate boilerplate

Andrewnt219 committed:
minor translation

Andrewnt219 committed:
minor translation

Andrewnt219 committed:
minor translation

vitokhangnguyen commented:
Chào bạn, xin lỗi vì để bạn chờ lâu quá. Mình vẫn đang review document của bạn nhé. Dạo này mình không có nhiều thời gian lắm. Mình sẽ cập nhật bàn dần. Mình  review 60% document rồi nha.

vitokhangnguyen committed:
Review up to line 163

Andrewnt219 committed:
update translation from line 163

Andrewnt219 committed:
Merge branch 'master' of https://github.com/gatsbyjs/gatsby-vi

Andrewnt219 committed:
reverse

Andrewnt219 committed:
Merge branch 'tutorial/part-eight' of https://github.com/Andrewnt219/gatsby-vi into tutorial/part-eight

Andrewnt219 committed:
update from line 163

Andrewnt219 commented:
> Chào bạn, xin lỗi vì để bạn chờ lâu quá. Mình vẫn đang review document của bạn nhé. Dạo này mình không có nhiều thời gian lắm. Mình sẽ cập nhật bàn dần. Mình review 60% document rồi nha.

Em mới update phần dịch từ 163 trở lên để sát nghĩa hơn ạ.
